apps/auto-merchant-bot/.env.example

```
@@ -0,0 +1,37 @@
+# caller's linked wallet against the ad's chain kind.
+
+# Sepolia admin private key (0x-prefixed, 32 bytes).
+EVM_ADMIN_PRIVATE_KEY=
+
+# Stellar admin ed25519 secret seed (S...).
+STELLAR_ADMIN_SECRET=
+
+# ── RPCs ────────────────────────────────────────────────────────────
+EVM_RPC_URL=
+STELLAR_RPC_URL=https://soroban-testnet.stellar.org
+STELLAR_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STELLAR_NETWORK_PASSPHRASE key should go before the STELLAR_RPC_URL key

(UnorderedKey)